cms/tests/test_rendering.py

```diff
@@ -40,6 +41,44 @@ def sample_plugin_context_processor(instance, placeholder, original_context):
     }
 
 
+
+@override_settings(
+    CMS_TEMPLATES=[
+        (TEMPLATE_NAME, TEMPLATE_NAME),
+        ('extra_context.html', 'extra_context.html')
+    ],
+)
+class RenderingEmptyTestCase(CMSTestCase):
+    def setUp(self):
+        super().setUp()
+        self.test_data = {
+            'title': u'RenderingTestCase-title',
```

Comment:
Does this need to be enforced as unicode, i'd avoid putting in any backwards compat if it's not needed. If it's absolutely need then fair enough.

Reply:
It was a copy and paste. I'll remove `u`. It doesn't really have any effect in python 3 anyway.